index.html

```diff
@@ -470,6 +470,14 @@ <h3 class="grouping-title">Methods</h3>
               value for <i>encoderOptions</i> between 0 and 1 (<i>encoderOptions</i>
               defaults to 0.92 otherwise).</p>
             </li>
+            <li>
+              <div>toBlob(mimeType, qualityArgument)</div>
```

Comment:
It might be worth noting that `qualityArgument` is currently only available in Chrome (desktop only) and Firefox ([caniuse](https://caniuse.com/#feat=mdn-api_htmlcanvaselement_toblob_image_quality)).